### ISSUE TITLE
--sign should be on by default when encrypting (and --verify on decrypt)

### ISSUE THREAD

**Issue by jacobian** [keybase/node-client#123]
Signing when encrypting, and verifying when decrypting, seem like pretty natural and logical defaults. The failure modes when you don't do those things can be severe, especially for less-security-aware users. I think many people don't fully get the ramifications of sending encrypted-but-unsigned messages and (worse) those of trusting an encrypted message without checking the signature.

It seems like the client should default to the option that's going to be best for most users and use cases, and that appears to me to be signing and verifying by default.

**Comment by SalusaSecondus** [comment]
I agree. The majority of people don't understand repudiation and will probably assume that since they received an encrypted message from Alice (over some insecure channel) that it must actually be from Alice. Signing will do the right thing for most people most of the time.
